### ISSUE TITLE
Drag and Drop a subtitle works except in ChromeCast

### ISSUE THREAD

**Issue by kripskroll** [Stremio/stremio-bugs#17]
Hi,

Subtitles are work perfectly when using Chromecasr except if you add your subtitle (.srt file) that you drag and drop in the player.
A new subtitle is present with name "custom" and it works fine in the player. But once we cast the stream, the subtitle is not seen. All default ones are working but not the customs one.

PS : Would be great to be able to use srt name instead of default "custom" name in the Player GUI for a subtitle drag and drop. If you add many you have many "Custom" and it is then impossible to know which one is each of them.

Thanks

**Comment by Ivshti** [comment]
Good observation! Will fix

**Comment by cheyde** [comment]
Yes, we need this fix :)

**Comment by miguelmendes** [comment]
Need this fix too :) gwork

**Comment by Ivshti** [comment]
Yeah, sorry for the slow progress, we have 2 developers in total including me and we're building an android app!

Will get on to that issue ASAP

**Comment by kripskroll** [comment]
No worries. That's fine
But why don't you open your code instead of partially hide it with your asar files compression ?

**Comment by jrnoir** [comment]
Still can't stream a drag and drop subtitle to Chromecast. Any idea of when it will be possible to do so? As non english speaker, this is a very important functionality to me and probably to a lot of people around the world as well.

**Comment by Ivshti** [comment]
@jrnoir @kripskroll @wormangel @CristianHeyde 

Beta 3.6.1 is being now uploaded, it should be finished in 10 minutes.

Please check it at http://beta.strem.io or simply toggle "Always use latest beta" in Settings and it would immediately auto-update

**Comment by ShinkoNet** [comment]
@Ivshti 
I'm using release 3.6.1. This problem still hasn't been fixed - dragging the .srt on a chromecast-playing video does nothing. The file loads on the stremio application (it says the file's name at the bottom, then disappears), but on my TV there aren't any subtitles shown.

**Comment by Ivshti** [comment]
actually, we haven't tested it that way; in our tests, we first drag the .srt and then cast; perhaps that's not always the real use case. Can you test it that way though?

**Comment by ShinkoNet** [comment]
I tried it numerous ways.
I'm trying to add subtitles onto a .mkv. I am using the OSX version.

When I drag the subtitle file onto the playing movie, it works fine. I click the Chromecast option and cast it to my TV. Here, the subtitles disappear.

If I decide to MUX the mkv and add the subtitle track that way (with a .ASS subtitle track) , it plays fine in VLC but not in streamio. When dragging a .SRT subtitle on the video, the movie crashes and stops playing. This is probably because the muxed .ass subtitle track is incompatible, right?

I could try muxing a .SRT file to the mkv and try again, but it seems that using airwave app plays the cast and subtitle fine so I'm using that for now.

**Comment by ShinkoNet** [comment]
Also, I should say that in the subtitle menu, when it loads after I drag an .SRT in, is blank with the + and - not doing anything, except for a "wrong subtitle" text and an on/off switch. Is this intended?

**Comment by Ivshti** [comment]
Not intended at all

Can you start Stremio in dev mode with ". --development" at the end of the
command line and give me a js stack trace. There should be an exception
when what you described happens

Otherwise Mkv muxed subtitles are not supported, and chromecasting is
transcoded to HLS

**Comment by Ivshti** [comment]
This should be re-opened as it's not working in latest versions

**Comment by parweb** [comment]
Any news on this ?
